src/main/java/net/trollyloki/murdermystery/game/Game.java

```diff
@@ -627,13 +649,25 @@ public void onEntityDamageByEntity(EntityDamageByEntityEvent event) {
                 }
                 // Handling for if none of the above items were used to damage
                 else if (player.getInventory().getItemInMainHand().getType() == Material.BAKED_POTATO) {
-                    player.getInventory().remove(Material.BAKED_POTATO);
-                    ((Player) event.getEntity()).getInventory().addItem(potato);
-                    // haha funny sound
-                    event.getEntity().getWorld().playSound(event.getEntity().getLocation(), Sound.BLOCK_FIRE_EXTINGUISH, 1, 1);
-                    // new potato victim
-                    potatoVictim = event.getEntity().getUniqueId();
-
+                	if (!potatoCooldown) {
+                		player.getInventory().remove(Material.BAKED_POTATO);
+                		((Player) event.getEntity()).getInventory().addItem(potato);
+                		// haha funny sound
+                		event.getEntity().getWorld().playSound(event.getEntity().getLocation(), Sound.BLOCK_FIRE_EXTINGUISH, 1, 1);
+                		// new potato victim
+                		potatoVictim = event.getEntity().getUniqueId();
+                		// particles!!!!
+                		event.getEntity().getWorld().spawnParticle(Particle.VILLAGER_ANGRY, event.getEntity().getLocation().add(0, 1, 0), 3, 0.3, 0, 0.3);
+                		// set potato cooldown and set bukkitrunnable to remove it in a second
+                		potatoCooldown = true;
+                		new BukkitRunnable() {
+							@Override
+							public void run() {
+								potatoCooldown = false;
+								
+							}
+                		}.runTaskLater(plugin, 20);
```

Comment:
Not a huge fan of using a BukkitRunnable for this cause it could still run after a game ends but since its only a second it should be fine

Reply:
oh, I see what you did with bow cooldowns, you made it work through the main game loop
i won't change my thing rn but if you want later you can